units/host_termination.go

```diff
@@ -344,6 +344,13 @@ func (j *hostTerminationJob) Run(ctx context.Context) {
 			"job_type": j.Type().Name,
 		}))
 		return
+	} else {
+		grip.Info(message.Fields{
+			"message": "host successfully terminated",
```

Comment:
is there any other information that would make this easier to consume in splunk? (like the distro id, for instance)